packages/victory-chart/src/index.d.ts

```diff
@@ -9,14 +9,23 @@ import {
   VictoryStyleInterface
 } from "victory-core";
 
+export type AxesType = {
```

Comment:
Does this type need to be exported? If you aren't referencing it beyond this file, I think you're ok to just declare the `type`

Reply:
I noticed there was a pattern of exporting types and interfaces from the d.ts files? It's been useful to be able to import types directly in demos